.github/workflows/codeforafrica-deploy-review-app.yml

```
@@ -0,0 +1,135 @@
+name: Code for Africa | Deploy | REVIEW APP
+
+on:
+  pull_request:
+    paths:
+      - "apps/codeforafrica/**"
+      - "Dockerfile.codeforafrica"
+      - ".github"
+concurrency:
+  group: "${{ github.workflow }} @ ${{ github.ref }}"
+  cancel-in-progress: false
+
+env:
+  DOKKU_REMOTE_BRANCH: "master"
+  DOKKU_REMOTE_URL: "ssh://azureuser@ui-1.dev.codeforafrica.org"
+  IMAGE_NAME: "codeforafrica/codeforafrica-dev"
```

```suggestion
  IMAGE_NAME: "codeforafrica/codeforafrica-ui"
```
They will just have different tags/versions.